### DISCUSSION TITLE
Installer alert

### DISCUSSION THREAD

**Discussion by discodancerstu** [petrleocompel/hikaxpro_hacs#112] [Q&A]
Hi all,

I keep getting the attached error, so alarm cant be armed.  Any ideas please?  Beginning to wish I hadn't bought this crappy system.....
![Screenshot 2024-04-29 162808](https://github.com/petrleocompel/hikaxpro_hacs/assets/123293641/580ffefe-8c5a-4e92-a157-7332516cfb33)

**Comment by petrleocompel** [comment]
I think this is really your personal opinion. 

I have no clue who installed your system. But I would not let anybody install it without fully giving me all permissions and removing him from the system. 

Sadly I have no clue what is your problem. I would look in WebUI as admin. Or in logs. Also we are focused on HomeAssistant integration.
